shell/browser/api/electron_api_session.cc

```diff
@@ -697,10 +698,19 @@ v8::Local<v8::Promise> Session::LoadExtension(
     return handle;
   }
 
+  int load_flags = extensions::Extension::FOLLOW_SYMLINKS_ANYWHERE;
+  gin_helper::Dictionary options;
+  if (args->GetNext(&options)) {
+    bool allow_file_access = false;
+    options.Get("allow_file_access", &allow_file_access);
```

Comment:
Camel case is more natural for Electron properties.
```suggestion
    options.Get("allowFileAccess", &allow_file_access);
```

Reply:
I think this approach is good and lends itself to further extensibility down the road.

Reply:
Done, thanks!